routes/api/users.js

```diff
@@ -134,26 +137,43 @@ router.put("/editItem", async (req, res) => {
     const modifiedItem = {
       name: req.body.name,
       description: req.body.description,
-      price: req.body.price
+      price: req.body.price,
     };
     //add items to document
     User.findOneAndUpdate(
       { username: req.user.username },
-      { $pull: { items: { _id: req.body.itemId } } }, { safe: true, upsert: true }).then(
-        User.findOneAndUpdate(
-          { username: req.user.username },
-          { $push: { items: modifiedItem } },
-          { safe: true, upsert: true, new: true, runValidators: true }
-        ).then(dbItems => {
+      { $pull: { items: { _id: req.body.itemId } } },
+      { safe: true, upsert: true }
+    ).then(
+      User.findOneAndUpdate(
+        { username: req.user.username },
+        { $push: { items: modifiedItem } },
+        { safe: true, upsert: true, new: true, runValidators: true }
+      )
+        .then((dbItems) => {
           res.json(dbItems);
-        }).catch(err => {
+        })
+        .catch((err) => {
           res.json(err);
-        }).catch(err => {
+        })
+        .catch((err) => {
           res.json(err);
-        }));
+        })
+    );
   } else {
-    res.status(404).json({ msg: 'NO SELLER LOGGED IN' });
+    res.status(404).json({ msg: "NO SELLER LOGGED IN" });
```

Comment:
my auto-formatter made the above changes.

changes made below are the main part of this PR